Runtime/Model/Metrics/UniqueEvent.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Backtrace.Unity.Json;
+using System.Collections.Generic;
+
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Backtrace.Unity.Tests.Runtime")]
+namespace Backtrace.Unity.Model.Metrics
+{
+    public sealed class UniqueEvent : EventAggregationBase
```

Comment:
When do we clear UniqueEvents?

Reply:
I see...we just update the timestamp of all unique events we are tracking, thank you for explanation @konraddysput